### PR TITLE
Removing requestAnimationFrame polyfill now we've dropped IE9.

### PR DESCRIPTION
**Pull request by sgomes**
@surma or @Garbee: PTAL!

### CODE REVIEW

**Comment by sgomes** [comment]
One sec, investigating build breakage.

**Comment by sgomes** [comment]
Looks like node doesn't support `requestAnimationFrame` and so the JS tests fail. Added a tiny shim for testing only.

**Comment by sgomes** [comment]
Good to review now :)

File: docs/_assets/customizer.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ function init() {
     mc.template = this.responseText;
     mc.highlightField('Indigo');
     mc.highlightField('Pink');
-    window.requestAnimationFrame(function() {
```
Comment (tleunen) [inline docs/_assets/customizer.js]: It's still a function from `window`, no reason to remove that ;)

Comment (sgomes) [inline docs/_assets/customizer.js]: That's true, this was a bit of an offtopic cleanup :)

**Comment by Garbee** [comment]
LGTM

+1 on dropping the window call. It isn't providing any benefit.

**Comment by tleunen** [comment]
It helps to know where it comes from :) And also prevent having an exception in file linting

**Comment by sgomes** [comment]
Thanks, @Garbee!

**Pull request merged**